ggmolvis/camera.py

```diff
@@ -93,6 +95,13 @@ def render(self,
         if composite_bg_rgba is not None:
            _set_compositor_bg(composite_bg_rgba)
 
+        if add_frame_numbers is not None:
+            img_path = _create_frame_image(frame_number=self.frame_number,
+                                           width=resolution[0],
+                                           height=resolution[1],
+                                           text=f"frame: {self.frame_number}")
+            _composit_frame_label(img_path=img_path)
```

Comment:
Note that this won't work for movies yet--some/all of the infrastructure may need to be migrated to `_update_frame()`, which is a bit annoying for design perhaps.